API key added in the beforeStore

oliviernolbert opened the pull request:
The API key from plugin options can now be set to the JGoogleEmbedMaps
class with the setKey() method.

This avoids error message from Google API when function to retrieve
latitude/longitude is called.

olivier-nolbert committed:
API key added in the beforeStore

The API key from plugin options can now be set to the JGoogleEmbedMaps
class with the setKey() method.

This avoids error message from Google API when function to retrieve
latitude/longitude is called.

PierreCorell reviewed (approved):
thank you, works as expected
closes #25

olivier-nolbert committed:
API Key Alternative added

An alternative API Key can be used on the OnStore event if first API
key has ‘HTTP referrers’ restrictions.

olivier-nolbert committed:
French translation added

sebastienheraud commented:
Hello,
Thanks for the PR and test.
Saba.
